capellambse/model/__init__.py

```diff
@@ -238,13 +250,31 @@ def __init__(
 
         if jupyter_untrusted is not None:
             warnings.warn(
-                (
-                    "The 'jupyter_untrusted' argument is no longer needed and"
-                    " will be removed soon. Please remove it from your calls."
-                ),
+                "The 'jupyter_untrusted' argument is no longer needed and"
+                " will be removed soon. Please remove it from your calls.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
Has nothing to do with the `diagram_cache` rename but still good catch. Mixing these things into one commit can be annoying in the future. Keep that in mind. Even though, I forget this from time to time,...

Reply:
What's the "good catch"? As far as I can see, the only thing that changed was some formatting (no parenthesis around the string anymore).

Reply:
I believe these formatting changes were made by running `black --preview capellambse/model/__init__.py` and not really by me.